readme.md

```diff
@@ -1,40 +1,46 @@
-# Material UI - Create React App example in TypeScript
+# Team 401 Copper Scout -  a First Robotics Competition Scouting App
 
 ## How to use
 
-Download the example [or clone the repo](https://github.com/mui/material-ui):
+[clone the repo](https://github.com/team401/scouting):
 
 <!-- #default-branch-switch -->
 
 ```bash
-curl https://codeload.github.com/mui/material-ui/tar.gz/master | tar -xz --strip=2 material-ui-master/examples/material-ui-cra-ts
-cd material-ui-cra-ts
+git clone https://github.com/team401/scouting
+cd scouting
 ```
 
-Install it and run:
+Add The Blue Alliance API Key ([free at](thebluealliance.com)) to data.tsx file
 
+Add Supabase URL and Anion key to ```./src/Supabase/.env.local```
```

Comment:
*Anon

Reply:
I'll update that and the .env file ( also shows anion )